pcsx2/CDVD/IsoFS/IsoFS.cpp

```diff
@@ -49,8 +50,9 @@ IsoDirectory::IsoDirectory(SectorSource& r)
 	uint i = 16;
 
 	m_fstype = FStype_ISO9660;
-
-	while( !done )
+	
+	bool loadFromDisc = g_Conf->CdvdSource != CDVD_SourceType::NoDisc;
+	while( !done && loadFromDisc)
```

Comment:
The error only appears on the first boot when NoDisc is the source. If you boot with another source and switch to NoDisc later, you won't see the message. I guess some other data related to disk sectors is getting left behind. Hope that clears it up.